libraries/ESP32/examples/Serial/onReceiveExample/onReceiveExample.ino

```diff
@@ -57,67 +62,52 @@
 
```

Comment:
Given that `Serial0` always exists, why not use that instead of mapping to UART0?

Reply:
Ok, let me just make sure it won't fail and then I'll change it.